device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsIotHubConnection.java

```diff
@@ -1202,10 +1209,18 @@ private void openAsync() throws TransportException
         }
 
         this.reactor = createReactor();
+
+        String runnerUniqueIdentifier = this.isMultiplexing
+                ? "Multiplexed" + this.transportUniqueIdentifier
+                : this.deviceClientConfig.getDeviceClientUniqueIdentifier();
+
+        String reactorRunnerPrefix = this.hostName + "-" + runnerUniqueIdentifier + "-" + "Cnx" + this.connectionId;
         ReactorRunner reactorRunner = new ReactorRunner(
             this.reactor,
             this.listener,
             this.connectionId,
+            reactorRunnerPrefix,
+            "OpenAsync",
```

Comment:
This reactor thread owns the connection, not just opening the connection. I'd consider getting rid of this "OpenAsync" postfix here.

Reply:
I see, Will do.